examples/UdpNtpClient/UdpNtpClient.ino

```diff
@@ -0,0 +1,174 @@
+/*
+
+ Udp NTP Client
+
+ Get the time from a Network Time Protocol (NTP) time server
+ Demonstrates use of UDP sendPacket and ReceivePacket
+ For more on NTP time servers and the messages needed to communicate with them,
+ see http://en.wikipedia.org/wiki/Network_Time_Protocol
+
+ created 4 Sep 2010
+ by Michael Margolis
+ modified 9 Apr 2012
+ by Tom Igoe
+ modified 02 Sept 2015
+ by Arturo Guadalupi
+
+ This code is in the public domain.
+
+ */
+
+#include <QNEthernet.h>
+
+using namespace qindesign::network;
+
+#define USING_DHCP    true
+
+#if !USING_DHCP
+  // Set the static IP address to use if the DHCP fails to assign
```

Comment:
I don't this comment accurately describes what happens. The variables aren't included if USING_DHCP is true.

Additionally, I'd add a comment requesting the user to assign valid values for their setup.

Reply:
Thanks for your comments.

This is !USING_DHCP, not USING_DHCP. To be used only for static IP.
User has to change them to match local network.